solr/server/solr/solr.xml

```diff
@@ -34,7 +34,6 @@
   <str name="allowPaths">${solr.allowPaths:}</str>
   <str name="allowUrls">${solr.allowUrls:}</str>
   <str name="hideStackTrace">${solr.hideStackTrace:false}</str>
-  <int name="indexSearcherExecutorThreads">${solr.searchThreads:4}</int>
```

Comment:
Removing the configuration so that the `DEFAULT_INDEX_SEARCHER_EXECUTOR_THREADS` from `NodeConfig.java` gets used.